src/ape/managers/networks.py

```diff
@@ -86,7 +86,9 @@ def get_provider_from_choice(
         provider_settings: Optional[Dict] = None,
     ) -> ProviderAPI:
         if network_choice is None:
-            return self.default["development"].get_provider(provider_settings=provider_settings)
+            return self.default_ecosystem["development"].get_provider(
```

Comment:
There is no `self.default`, I wonder why mypy didn't complain
So this is like a bug fix to an unreleased piece of work I am sneaking in...